paper/src/main/java/com/denizenscript/denizen/paper/events/PlayerClicksFakeEntityScriptEvent.java

```diff
@@ -0,0 +1,99 @@
+package com.denizenscript.denizen.paper.events;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.PlayerTag;
+import com.denizenscript.denizen.utilities.entity.FakeEntity;
+import com.denizenscript.denizen.utilities.implementation.BukkitScriptEntryData;
+import com.denizenscript.denizencore.events.ScriptEvent;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntryData;
+import com.destroystokyo.paper.event.player.PlayerUseUnknownEntityEvent;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+
+public class PlayerClicksFakeEntityScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // player clicks fake entity
+    // player (right/left) clicks fake entity
+    //
+    // @Regex ^on player ([^\s]+ )?clicks fake entity$
+    //
+    // @Switch in:<area> to only process the event if it occurred within a specified area.
+    //
+    // @Plugin Paper
+    //
+    // @Triggers when a player clicks a fake entity, one that is only shown to the player and not tracked by the server.
+    //
+    // @Context
+    // <context.entity> returns the EntityTag of the entity that was clicked. Note that this entity is not being tracked by the server, so many operations may not be possible on it.

```

Comment:
This should be documented as potentially null, or the event changed to not fire if it would be null.